### DISCUSSION TITLE
The last of us

### DISCUSSION THREAD

**Discussion by evefevrier** [integration2/quizTIM2023#53] [Q&A]
Commentaires sur ce quiz.

**Accès**
- Serveur de développement (protégé) :  https://timunix3.csfoy.ca/~quiz-TIM/ARTADestrokzZ/index.html
- Dépôt Github (privé pour le moment): https://github.com/ARTADestrokzZ/quiz-tim
- Github.io :

**Comment by ARTADestrokzZ** [comment]
@evefevrier 
Euh ça veut tu dire que vous ne pouvez pas le voir ?
Dans ce cas nous pourrions essayer d’arranger ça. Je ne sais pas comment faire.

**Comment by evefevrier** [comment]
@ARTADestrokzZ 
Je peux voir et faire ton quiz d'un bout à l'autre, ça va très bien.
Il y a une seule chose qui me dérange vraiment, ce sont les rétroactions.
Serait-il possible d'enlever les phrases "Tu es nul" et "Tu es le ou la
meilleure" ?
Je pense qu'il serait plus agréable de faire le quiz sans cela.

Le sam. 13 mai 2023 à 15:32, ARTADestrokzZ ***@***.***> a
écrit :

> @evefevrier <https://github.com/evefevrier>
> Euh ça veut tu dire que vous ne pouvez pas le voir ?
> Dans ce cas nous pourrions essayer d’arranger ça. Je ne sais pas comment
> faire.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/integration2/quizTIM2023/discussions/53#discussioncomment-5893696>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABGZJ5ANJJP6VA46EOPNKRTXF7OV5ANCNFSM6AAAAAAX5BMLL4>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Eve Février
Enseignante en Techniques d'intégration multimédia
Cégep de Sainte-Foy
Tél.(418)659-6600poste6655
